mono/utils/mono-threads-state-machine.c

```diff
@@ -85,10 +85,11 @@ check_thread_state (MonoThreadInfo* info)
 }
 
 static inline void
-trace_state_change (const char *transition, MonoThreadInfo *info, int cur_raw_state, int next_state, int suspend_count_delta)
+trace_state_change_with_func (const char *transition, MonoThreadInfo *info, int cur_raw_state, int next_state, int suspend_count_delta, const char *func)
 {
 	check_thread_state (info);
-	THREADS_STATE_MACHINE_DEBUG ("[%s][%p] %s -> %s (%d -> %d)\n",
+	THREADS_STATE_MACHINE_DEBUG ("%s [%s][%p] %s -> %s (%d -> %d)\n",
```

Comment:
Add the funct to the end. Having the same prefix makes it easier to read a lot of logs.

Reply:
ok